src/scope/VariableMarker.js

```diff
@@ -62,16 +68,17 @@ class VariableMarker {
    * @param  {String} varName
    */
   markModified(varName) {
-    const blockVar = this.getScope().findBlockScoped(varName);
-    if (blockVar) {
-      blockVar.markModified();
+    const blockVars = this.getScope().findBlockScoped(varName);
+    if (blockVars) {
```

Comment:
Here we also deal with the `false` return value, which the `findBlockScoped()` method shouldn't return.